docker/develop-ci/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM cyclus/cyclus:latest
+
+COPY . /cycamore
+WORKDIR /cycamore
+RUN ./install.py
+
```

Comment:
The only difference should be on the cyclus/cyclus docker container tag we are using...